### PR TITLE
Adds SSL on devmode (uses Plya's hack to create a keystore on the fly)

### PR DESCRIPTION
**Pull request by ignasi35**
Using play docs and sample apps put together HTTPS for dev mode in play. Haven't tested the app though (seen it running and listening on 9443)

### CODE REVIEW

**Comment by ignasi35** [comment]
This PR includes a `ssl-play` script (taken from a [sample app](https://www.playframework.com/documentation/2.6.x/AkkaHttpServer#Using-HTTP/2-in-run-mode) mentioned in the [docs](https://www.playframework.com/documentation/2.6.x/AkkaHttpServer#Using-HTTP/2-in-run-mode)) which _replaces_ `sbt`. Once this PR is merged, there'll be three options:

1. `sbt` starts sbt without enabling ssl
2. `./ssl-play` starts sbt enabling ssl
3. gradle - based (not sure what the command is)

File: ssl-play [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+# Enable this if you need TLS debugging
+# http://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/JSSERefGuide.html#Debug
+#JVM_OPTIONS="$JVM_OPTIONS -Djavax.net.debug=ssl:handshake"
+
+# Change this if you need X.509 certificate debugging
+# http://docs.oracle.com/javase/8/docs/technotes/guides/security/troubleshooting-security.html
+#JVM_OPTIONS="$JVM_OPTIONS -Djava.security.debug=certpath:x509:ocsp"
+
+
+export AGENT=$(pwd)/$(find target -name 'jetty-alpn-agent-*.jar' | head -1)
```
Comment (raboof) [inline ssl-play:59]: The agent jar isn't there for me - how does it get there? I'd be OK with adding it in the build.sbt instead

Comment (raboof) [reply ssl-play:59]: Ah but play doesn't run forked so it needs to be in the SBT_OPTS :+1:.

The question remains how it ended up in the target dir for you :)

**Review by raboof — APPROVED** [review]
One question about the agent but other than that LGTM!

File: ssl-play (hunk 1 above)
Comment (raboof) [reply ssl-play:59]: ah, `sbt stage`

**Comment by ignasi35** [comment]
Pff, just seen @raboof's comments. My gmail filters are wrong putting too much stuff into a single folder.

**Pull request merged**